golem-router/golem-services.conf.template

```diff
@@ -0,0 +1,36 @@
+# This configuration is geared to use with docker-compose examples
+events {
+}
+
+http {
+    # For docker we need this for service discovery in docker network
+    resolver 127.0.0.11;
+
+    server {
+        listen 80;
+        server_name localhost;
+
+        location ~ /v2/templates/[^/]+/workers/[^/]+/connect$ {
+            proxy_pass http://$GOLEM_WORKER_SERVICE_HOST:$GOLEM_WORKER_SERVICE_PORT;
+            proxy_http_version 1.1;
+            proxy_set_header Upgrade "websocket";
+            proxy_set_header Connection "upgrade";
+        }
+
+        location /v1/api {
+            proxy_pass http://$GOLEM_WORKER_SERVICE_HOST:$GOLEM_WORKER_SERVICE_PORT;
+        }
+
+        location ~ /v2/templates/[^/]+/workers(.*)$ {
+            proxy_pass http://$GOLEM_WORKER_SERVICE_HOST:$GOLEM_WORKER_SERVICE_PORT;
+        }
+
+        location /v2/templates {
+            proxy_pass http://$GOLEM_TEMPLATE_SERVICE_HOST:$GOLEM_TEMPLATE_SERVICE_PORT;
+        }
+
+        location / {
+            proxy_pass http://$GOLEM_TEMPLATE_SERVICE_HOST:$GOLEM_TEMPLATE_SERVICE_PORT;
+        }
```

Comment:
i think it should be possible to add and serve openapi yaml by nginx

Reply:
We can do it later. We have the yaml generated in the yaml directory anyway as before